src/components/Attachment/Attachment.tsx

```diff
@@ -169,6 +185,21 @@ class Attachment extends UIComponent<Extendable<AttachmentProps>, any> {
       </ElementType>
     )
   }
+
+  protected actionHandlers: AccessibilityActionHandlers = {
+    performClick: event => this.handleKeyboardClick(event),
+  }
+
+  private handleKeyboardClick = e => {
+    if (e.keyCode === keyboardKey.Spacebar) {
```

Comment:
why is preventDefault called only for spacebar? components should not have any knowledge about the key pressed